config.cfg

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+set TOGGLE_CONSOLE `
```

Comment:
cool stuff :)
do you think it's better placing this file to `dist/`? also, it's not really a `.cfg` with the unix-cfg-like-syntax, so that file suffix is maybe inappropriate (`.oac`, `.oacvar`, ...?)